benchmarks/benchmarks/infinite_sheds.py

```diff
@@ -0,0 +1,100 @@
+"""
+ASV benchmarks for infinite_sheds.py
+"""
+
+import numpy as np
+import pandas as pd
+from pvlib.bifacial import infinite_sheds
+from pvlib import location, tracking
+
+
+class InfiniteSheds:
+
+    def setup(self):
+        self.times = pd.date_range(start='20180601', freq='1min',
+                                   periods=8760*25)
+        self.location = location.Location(40, -80)
+        self.solar_position = self.location.get_solarposition(self.times)
+        self.clearsky_irradiance = self.location.get_clearsky(self.times)
```

Comment:
Maybe worth using a faster solar position algorithm here, and passing the result into `get_clearsky` to avoid a redundant calculation?

Reply:
the setup code doesn't contribute to the benchmark timing and I think it's now fast enough overall to keep it simple.